use-cases.md

```diff
@@ -0,0 +1,5 @@
+ - real estate registries
+ - elections
```

Comment:
I was thinking about elections yesterday, it very apt there

Reply:
And I see it has huge case in Judicial system also, laws, judgement, court decree